src/pincode/Pincode.tsx

```diff
@@ -64,14 +58,6 @@ function Pincode({
       <View style={styles.spacer} />
       {!errorText && <Text style={styles.title}>{title || ' '}</Text>}
       {!!errorText && <Text style={styles.error}>{errorText}</Text>}
-      <ScrollView contentContainerStyle={styles.contentContainer}>
-        {showGuidedOnboarding && (
```

Comment:
wouldn't it be easier to just lift this `{showGuidedOnboarding && ...` part up one level? So turning the feature flag off hides the whole scroll view. Then we wouldn't need to mess with the migrations

Reply:
@cajubelt I am inclined to revert this then reevaluate the fixed version of https://github.com/valora-inc/wallet/pull/2817 when it is ready.